docs/QuickStart.md

```diff
@@ -237,15 +276,8 @@ POST to ```v1/subjects/USA/child``` to create a
     ]
 }
 ```
-### Coming Soon: Auth Tokens
-
-Refocus will soon also allow you to use auth tokens to add an additional layer of security on the platform. Stay tuned!
-
-### Create your own lens
-//TODO point at ldk
-
-# Thank You!
-Congrats on the deployment of your Refocus! If you have any questions or feedback, please reach out to us.
+### Authorization Tokens Are Coming Soon!
 
+Soon you can use authorization tokens to add an additional layer of security on the platform. Stay tuned!
```

Comment:
I would rather leave this out of the quickstart. Security setup should be its own document or be part of the quickstart api calls